pkg/apis/options/sessions.go

```diff
@@ -28,6 +28,7 @@ type RedisStoreOptions struct {
 	SentinelPassword       string   `flag:"redis-sentinel-password" cfg:"redis_sentinel_password"`
 	SentinelMasterName     string   `flag:"redis-sentinel-master-name" cfg:"redis_sentinel_master_name"`
 	SentinelConnectionURLs []string `flag:"redis-sentinel-connection-urls" cfg:"redis_sentinel_connection_urls"`
+	SentinelDB             int      `flag:"redis-sentinel-db" cfg:"redis_sentinel_db"`
```

Comment:
just so we have the some order. i would like to have the `SentinelDB` above the `SentinelPassword` like you did with the flag.

Reply:
I have ordered this